tools/check_consistency.py

```
@@ -0,0 +1,434 @@
+                with open(yml_file) as f:
+                    data = yaml.safe_load(f)
+                if data and "swap" in data:
+                    for term, alt in data["swap"].items():
+                        # Strip YAML regex artifacts
+                        clean_term = re.sub(r"[\\()]", "", term).lower().strip()
+                        # Remove regex quantifiers like (?:ed|ing)?
+                        clean_term = re.sub(r"\?\:", "", clean_term)
+                        clean_term = re.sub(r"[?+*]", "", clean_term)
+                        vale_terms[clean_term] = alt.lower() if isinstance(alt, str) else str(alt).lower()
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)